recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 1
```

Comment:
I think there's no need to increase the build number, so this should be reverted.

Reply:
@conda-forge/core, is it necessary to increase the build number to add support for aarch64?

Reply:
No, the existing non-aarch64 packages do not need to be rebuilt so there is no need to change the build number.  That said, bumping the build number is fine, it just means a few new packages are created that are not absolutely needed.

Reply:
ok, I will de-bump and submit a new pr.